<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(27, 39, 50, 255),stop:1 rgba(47, 53, 74, 255));
	color: #000000;
background-repeat: no-repeat; 
background-position: center;}</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1210</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(27, 39, 50, 255),stop:1 rgba(47, 53, 74, 255));
	color: #000000;
background-repeat: no-repeat; 
background-position: center;}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>581</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 39pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Insert Car Information</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>670</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
color: rgb(0, 0, 0);
background-color: rgb(0, 255, 203);
font: 75 25pt &quot;URW Bookman L&quot;;
</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="model">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>370</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu Mono&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="color">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>450</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu Mono&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>360</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>440</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="no">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>200</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pl">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>290</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu Mono&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>280</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Plate Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu Mono&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="make">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>520</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Ubuntu Mono&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>510</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(238, 238, 236)
</string>
     </property>
     <property name="text">
      <string>Make</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
color: rgb(0, 0, 0);
background-color: rgb(0, 255, 203);
font: 75 25pt &quot;URW Bookman L&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="error1">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>610</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;URW Bookman L&quot;;
color:rgb(239, 41, 41)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
